app/samples/src/main/java/com/sygic/samples/search/viewmodels/SearchFromBrowseMapActivityViewModel.kt

```diff
@@ -24,17 +24,49 @@
 
 package com.sygic.samples.search.viewmodels
 
-import android.util.Log
 import androidx.fragment.app.Fragment
+import androidx.lifecycle.LiveData
 import androidx.lifecycle.ViewModel
 import com.sygic.maps.module.common.provider.ModuleConnectionProvider
 import com.sygic.maps.module.search.SearchFragment
+import com.sygic.maps.uikit.viewmodels.common.extensions.loadDetails
+import com.sygic.maps.uikit.views.common.extensions.asSingleEvent
+import com.sygic.maps.uikit.views.common.livedata.SingleLiveEvent
+import com.sygic.sdk.search.MapSearchResult
+import com.sygic.sdk.search.Search
 import com.sygic.sdk.search.SearchResult
+import com.sygic.sdk.search.detail.*
 
 class SearchFromBrowseMapActivityViewModel : ViewModel(), ModuleConnectionProvider {
 
+    val showToastObservable: LiveData<String> = SingleLiveEvent()
+
     private val callback: ((searchResultList: List<SearchResult>) -> Unit) = { searchResultList ->
-        searchResultList.forEach { Log.d("Test", it.type.toString()) } //todo: MS-5213 add it to the map?
```

Comment:
budeme zobrazovat viac toastov? to pridavanie z `todo` nerobime?

Reply:
upravím na jeden toast